site/content/recipes/perform_code_review.md

```diff
@@ -2,15 +2,36 @@
 
 ## Code author communicates code and context
 
-1. Author selects an appropriate segment of code
-    - Pick a snippet you estimate can be reivewed in about 30 minutes.
-    - We suggest picking a piece of code that is 50-100 lines.
-    - If the code you plan to review is larger than that, agree on a strategy to sequentially go through smaller segments, or prioritise the most important segment
-    - Choice is helped by objectvies from MAO stage.
-      - TODO add examples: e.g. if coding style is reviewed, pick recently added class or method. The more representative of the codebase the snippet is, the more widely relevant the feedback.
-      - Author should be familiar with the details of the chosen snippet. The main idea here is to come prepared.
-
-It's a good idea to share the code to review with the reviewer ahead
+The first step initiating the actual code review phase is for the code author to
+select a segment of their code to be reviewed. This sounds obvious and perhaps
+even easy, but it can be quite a tricky task. The success and impact of the code
```

Comment:
I understand you want to motivate readers by saying choosing the code under review is harder than it sounds, btu as a read er I'm left with the questions: "what is hard about it?". Maybe, _maybe_ skip that sentence alltogether and jump straight to the discussion of how important this choice is for the code review (next sentence)?

Reply:
+1 agree with what @tlestang says here. I also like the concrete examples in the second chunk below, but I think you could communicate roughly the same thing by skipping the sentence about how "this sounds obvious" and instead just saying:

> The first step initiating the actual code review phase is for the code author to select a segment of their code to be reviewed. Your choice should address the objectives set at the [Meet and Agree on Objectives step](../meet_and_agree_on_objectives). ...

Reply:
Applied patch in separate comment to address this: https://github.com/ResearchCodeReviewCommunity/dev-review/pull/55/files/3286ada701535fdd396ff9ae187a2d0937cda0e3#r833547419